tools/benchmarks/llm_eval_harness/meta_eval/prepare_meta_eval.py

```diff
@@ -134,6 +134,40 @@ def change_yaml(args, base_name):
                         "WORK_DIR", str(yaml_dir)
                     )
                 )
+    # 3.2 evals dataset has a differents set of tasks from 3.1
+    # Update tasks in meta_pretrain.yaml
+    with open(args.template_dir + "/meta_pretrain.yaml", "r") as yaml_file:
+        meta_pretrain = yaml.safe_load(yaml_file)
+    if args.evals_dataset in [
+        "meta-llama/Llama-3.1-8B-evals",
+        "meta-llama/Llama-3.1-70B-evals",
+        "meta-llama/Llama-3.1-405B-evals",
+    ]:
+        meta_pretrain["task"] = ["meta_bbh", "meta_mmlu_pro_pretrain"]
+    elif args.evals_dataset in [
+        "meta-llama/Llama-3.2-1B-evals",
+        "meta-llama/Llama-3.2-3B-evals",
+    ]:
+        meta_pretrain["task"] = ["meta_mmlu"]
+    with open(args.work_dir + "/meta_pretrain.yaml", "w") as yaml_file:
+        yaml.dump(meta_pretrain, yaml_file)
+    
+    # Update tasks in meta_instruct.yaml
+    with open(args.template_dir + "/meta_instruct.yaml", "r") as yaml_file:
+        meta_instruct = yaml.safe_load(yaml_file)
+    if args.evals_dataset in [
+        "meta-llama/Llama-3.1-8B-Instruct-evals",
+        "meta-llama/Llama-3.1-70B-Instruct-evals",
+        "meta-llama/Llama-3.1-405B-Instruct-evals",
+    ]:
+        meta_instruct["task"] = ["meta_ifeval", "meta_math_hard", "meta_gpqa", "meta_mmlu_pro_instruct"]
+    elif args.evals_dataset in [
+        "meta-llama/Llama-3.2-1B-Instruct-evals",
+        "meta-llama/Llama-3.2-3B-Instruct-evals",
+    ]:
```

Comment:
I'm thinking of introducing constants:
`LLAMA_3_1_INSTRUCT_EVALS`, `LLAMA_3_1_PRETRAIN_EVALS`,
`LLAMA_3_2_INSTRUCT_EVALS`, `LLAMA_3_2_PRETRAIN_EVALS`

since we're duplicating these strings in a lot of places.

wdyt?